lib/utils.js

```diff
@@ -64,7 +64,8 @@ const getBabelOptions = function loadBabelOptions(filename, options = {}) {
 
 const getTsJestConfig = function getTsJestConfig(config) {
   const tr = createTransformer()
-  return tr.configsFor(config)
+  const { typescript } = tr.configsFor(config)
+  return { compilerOptions: typescript.options }
```

Comment:
Or we could do something like the this to make sure we dont depend on `ts-jest` to fetch tsConfig options when users do not have ts-jest as their configuration.

```
const getTsJestConfig = function getTsJestConfig(config = {}) {
  const vueTsConfig = getVueJestConfig(config)
  if (typeof vueTsConfig.tsConfig === 'string') {
    return getTsConfigJson(vueTsConfig.tsConfig)
  } else if (typeof vueTsConfig.tsConfig === 'object') {
    return vueTsConfig.tsConfig
  } else if (config.globals && config.globals['ts-jest']) {
    const tr = createTransformer()
    const {
      typescript: { options: compilerOptions }
    } = tr.configsFor(config)
    return { compilerOptions }
  } else {
    // auto pick tsconfig from the root folder
    return getTsConfigJson()
  }
}

const getTsConfigJson = function getTsConfigJson(
  pathToConfig = path.resolve(process.cwd(), './tsconfig.json')
) {
  const configJson = ts.parseConfigFileTextToJson(
    pathToConfig,
    ts.sys.readFile(pathToConfig)
  )
  const { options: compilerOptions } = ts.parseJsonConfigFileContent(
    configJson.config,
    ts.sys,
    ts.getDirectoryPath(pathToConfig),
    {},
    pathToConfig
  )
  return { compilerOptions }
}
```

Reply:
@eddyerburgh thoughts?

Reply:
For most use cases it would be fine to compile with Babel, it would also simplify the code. The only problem would be if users use const enums, namespaces, or `export =` or `import =`. 

I haven't seen anyone using those features in components, so I think we can go ahead with using Babel for compiling TypeScript.

Reply:
Since we added ts-jest, Jest added native support for TypeScript in v24 (which uses Babel), so I think we are fine to remove ts-jest and just use Babel.

Reply:
Can you please make this change in a future PR